cbmc_viewer/make_byteop.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# -*- mode: python-mode -*-
+
+"""List CBMC Byte Extract/Update metrics."""
+
+
+import argparse
+import sys
+
+from cbmc_viewer import byteopt
+from cbmc_viewer import optionst
+
+def create_parser():
+    """Command line parser."""
+
+    parser = argparse.ArgumentParser(
+        description="List CBMC byte extract and update metrics."
+    )
+
+    optionst.byteop(parser)
+    optionst.srcdir(parser)
+    optionst.reportdir(parser)
+
+    optionst.log(parser)
+
+    return parser
+
+################################################################
+
+def main():
+    """List CBMC byte extract and update metrics."""
+
+    args = create_parser().parse_args()
+    args = optionst.defaults(args)
+
+    try:
+        byteop = byteopt.ByteOpSummary(args.byteop,
+                                       args.srcdir)
+        byteop.dump(outdir=args.reportdir)
```

Comment:
This is dumping html.  All other make_TYPE scripts dump json in the json directory.  This appears to be creating the html output of the report.